### PR TITLE
apmelasticsearch: don't query cluster name

### PR DESCRIPTION
**Pull request by axw**
See https://github.com/elastic/apm/pull/688

### CODE REVIEW

**Review by marclop — APPROVED** [review]
Nice!

**Comment by axw** [comment]
I'll merge this (after fixing the integration test) once the spec PR change is fully approved and merged.

**Comment by axw** [comment]
Actually I'll merge this without waiting to avoid releasing with something that'll probably be taken away anyway.

**Pull request merged**
